backend.native/compiler/ir/backend.native/src/org/jetbrains/kotlin/backend/konan/ToplevelPhases.kt

```diff
@@ -384,6 +380,14 @@ internal val dependenciesLowerPhase = SameTypeNamedPhaseWrapper(
                             input.files.clear()
                         }
 
+                // Lower all referenced inline functions from cached libraries.
+                if (context.irFilesForInlineFunctions.isNotEmpty()) {
+                    input.files += context.irFilesForInlineFunctions.values
+                    allLoweringsPhase.invoke(phaseConfig, phaserState, context, input)
```

Comment:
What happens if it produces more `irFilesForInlineFunctions` or adds declarations to existing ones?